test/functional/api/buffer_spec.lua

```diff
@@ -301,37 +301,37 @@ describe('api/buf', function()
     end)
   end)
 
-  describe('get_offset_for_line', function()
-    local get_offset_for_line = curbufmeths.get_offset_for_line
+  describe('get_offset', function()
+    local get_offset = curbufmeths.get_offset
     it('works', function()
       curbufmeths.set_lines(0,-1,true,{'Some','exa\000mple', '', 'text'})
```

Comment:
would be helpful to include `\r\n` and `\r` somewhere in there

Reply:
Literal `'\r\n'` is not possible, but I could do `\r` in the middle of a line and at the end of a line.